unfetter-discover/stix
# Issue 994 1

## problem
category and capabilities should have a relationship
is_baseline is not longer relevant, as assessment_set will always be a baseline

## changes
* create a string relationship from category to capability via the category name.  simplistic we may need to expand later
* removed is_baseline
* updated diagram

## test
verify travis
* pull this feature branch in the stix project
* run `cd unfetter-ui && npm link ../stix`
* verify the ui `npm run test && npm run lint && npm run build:dev`
